gRPC/Client Examples/RealtimeGraphing/BICManager.cs

```diff
@@ -372,7 +440,10 @@ private bool validBufferOrderCheck(Google.Protobuf.Collections.RepeatedField<Neu
         /// <returns>Task completion information</returns>
         async Task neuralMonitorTaskAsync()
         {
-            var stream = deviceClient.bicNeuralStream(new bicNeuralSetStreamingEnable() { DeviceAddress = DeviceName, Enable = true, BufferSize = 100, MaxInterpolationPoints = 10, AmplificationFactor = RecordingAmplificationFactor.Amplification395DB, RefChannels = { 31 }, UseGroundReference = true });
+            // Sytem.InvalidOperationException- Shutdown has already been called
```

Comment:
if this is commented out does it need to be cleaned up?

Reply:
Removed extra comments